cmd/crc/cmd/delete.go

```diff
@@ -44,10 +48,6 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 		}
 	}
 
-	if err := checkIfMachineMissing(client); err != nil {
-		return false, err
-	}
-
```

Comment:
You were right when pointing out that we want `crc delete --clear-cache` to work without a preexisting machine, this was fixed once in `Issue #1102 Make clear-cache work when there is no existing crc vm`

Reply:
It's weird to have clear-cache flag under delete and not cleanup 🤔

Reply:
`delete --clear-cache` was probably added before `cleanup`, but good point.